packages/component/src/Dictation.js

```diff
@@ -50,7 +50,7 @@ const Dictation = ({ emitTypingIndicator, onError }) => {
         if (transcript) {
           setSendBox(transcript);
           submitSendBox('speech');
-          setShouldSpeakIncomingActivity();
+          setShouldSpeakIncomingActivity(true);
```

Comment:
Why is this change necessary?

Reply:
It was called `startSpeakActivity` and `stopSpeakActivity`. And now `shouldSpeakIncomingActivity`. We should always call it with `true`/`false`.

And I found test failed when I forget to set it to `true`.